controllers/storage.go

```diff
@@ -18,7 +18,13 @@ package controllers
 
 import (
 	"context"
+	"encoding/base64"
+	"fmt"
+	"net/http"
 
+	"github.com/go-logr/logr"
+	minio "github.com/minio/minio-go/v7"
```

Comment:
`minio` alias is redundant here you can remove it

Reply:
DSP/KFP still define the alias this way:
https://github.com/opendatahub-io/data-science-pipelines/blob/master/backend/src/apiserver/storage/object_store.go#L24

Reply:
updated =/